packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -26,15 +26,19 @@ const OfferBox: React.FC<OfferBoxProps> = ({ offer, selected, onChange }: OfferB
 
   const getFreeTrialText = (offer: Offer) => {
     if (offer.freeDays) {
-      return t('choose_offer.benefits.first_days_free', { count: offer.freeDays });
+      return offer.freeDays === 1
+        ? t('choose_offer.benefits.first_days_free_one')
+        : t('choose_offer.benefits.first_days_free_other', { count: offer.freeDays });
     } else if (offer.freePeriods) {
-      // t('periods.day', { count })
-      // t('periods.week', { count })
-      // t('periods.month', { count })
-      // t('periods.year', { count })
-      const period = t(`periods.${offer.period}`, { count: offer.freePeriods });
+      // t('periods.day_one') = 'day' or t('periods.day_other') = 'days'
```

Comment:
To enable plurals, you can fake a `count` property in the translate options

// t('periods.day', { count })

Reply:
That's strange, we've updated this part just the way you're describing. When I check out the `release-candidate` branch I see this:
![image](https://github.com/user-attachments/assets/8c5b17b6-f02e-4b7a-92fd-46721fd648d3)

Reply:
This is some strange diffing behavior by GitHub, the change you're commenting is the exact reversal of what is being displayed. In any case, this is just a comment. A no longer existing version of a comment, but a comment nonetheless. The actual code does utilize the `count` property.

Reply:
This is something we've addressed in the following PR:
https://github.com/jwplayer/ott-web-app/pull/683

The strange thing is that that PR is supposed to be included in this one, and these changes you're commenting on are not the final ones that we're supposed to see here.

Reply:
@mirovladimitrovski I didn't address commented i18n invocations.